java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -252,15 +262,17 @@ public MantaClient(final ConfigContext config,
 
         this.uriSigner = new UriSigner(this.config, keyPair, signer);
 
+        this.beanSupervisor = new MantaMBeanSupervisor();
```

Comment:
I'm unsure how this connects to the fork-join related changes.

Reply:
I messed up while fixing the merge conflict that was present this morning, there's now an extra assignment that needs to be removed.